builtin/providers/aws/resource_aws_route_table.go

```diff
@@ -301,7 +302,19 @@ func resourceAwsRouteTableUpdate(d *schema.ResourceData, meta interface{}) error
 			}
 
 			log.Printf("[INFO] Creating route for %s: %#v", d.Id(), opts)
-			if _, err := conn.CreateRoute(&opts); err != nil {
+			err := resource.Retry(1*time.Minute, func() *resource.RetryError {
+				_, err := conn.CreateRoute(&opts)
+				if err != nil {
+					if awsErr, ok := err.(awserr.Error); ok {
+						if awsErr.Code() == "InvalidRouteTableID.NotFound" && strings.Contains(awsErr.Message(), "does not exist") {
```

Comment:
Do we need the check for "does not exist" here? Seems like `InvalidRouteTableID.NotFound` should cover this case.